packages/react-router-dom/__tests__/link-click-test.tsx

```diff
@@ -138,7 +138,6 @@ describe("A <Link> click", () => {
           <div>
             <h1>Home</h1>
             <Link
-              reloadDocument
```

Comment:
Shouldn't have this on this `Link` since the external URL should be what forces it to behave as a full-document link click